### ISSUE TITLE
Feature request: `order` kwarg for `size`

### ISSUE THREAD

**Issue by asinghvi17** [rafaqz/DimensionalData.jl#886]
It would be nice if this worked:
```julia
resolution = size(earth_ras; order = (X, Y))
```
instead, we currently have to do:
```julia
resolution = length.(dims(earth_ras, (X, Y)))
```

and `order` seems to be a standard kwarg in DimPoints etc.

**Comment by rafaqz** [comment]
Yes that seems nice. I guess `axes` and maybe a few other methods will need it too to be consistent

**Comment by rafaqz** [comment]
Probably it would just be `size(earth_ras, (X, Y))` as `size(A, X)` already works. I think we don't have it because Base doesn't have `size(A, (1, 3))`.
